src/Symfony/Component/Notifier/Bridge/OvhCloud/OvhCloudTransport.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\OvhCloud;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Thomas Ferney <thomas.ferney@gmail.com>
+ *
+ * @experimental in 5.1
+ */
+final class OvhCloudTransport extends AbstractTransport
+{
+    private $endpoints = [
+        'ovh-eu' => 'https://eu.api.ovh.com/1.0',
+        'ovh-ca' => 'https://ca.api.ovh.com/1.0',
+        'ovh-us' => 'https://api.us.ovhcloud.com/1.0',
+        'kimsufi-eu' => 'https://eu.api.kimsufi.com/1.0',
+        'kimsufi-ca' => 'https://ca.api.kimsufi.com/1.0',
+        'soyoustart-eu' => 'https://eu.api.soyoustart.com/1.0',
+        'soyoustart-ca' => 'https://ca.api.soyoustart.com/1.0',
+        'runabove-ca' => 'https://api.runabove.com/1.0',
+    ];
+
+    private $applicationKey;
+    private $applicationSecret;
+    private $consumerKey;
+    private $serviceName;
+    private $timeDelta;
+
+    public function __construct(string $applicationKey, string $applicationSecret, string $consumerKey, string $serviceName, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->applicationKey = $applicationKey;
+        $this->applicationSecret = $applicationSecret;
+        $this->consumerKey = $consumerKey;
+        $this->serviceName = $serviceName;
+
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function setHostByEndpoint(?string $endpoint): self
+    {
+        $this->host = $this->endpoints[$endpoint] ?: self::HOST;
+
+        return $this;
+    }
+
+    public function __toString(): string
+    {
+        return sprintf('ovhcloud://%s?consumer_key=%s&service_name=%s', $this->getEndpoint(), $this->consumerKey, $this->serviceName);
```

Comment:
Could it be just "ovh" instead of "ovhcloud"?

Reply:
Do you think I should rename all the bridge or just this ? I named it that because the new company name is ovhcloud now.

Reply:
If this is the name of the company, keep it like this.